contracts/lupi.sol

```diff
@@ -3,9 +3,14 @@ pragma solidity ^0.4.11;
 import "./Owned.sol";
 
 contract lupi is owned {
```

Comment:
Should be `Lupi` based on naming conventions: http://solidity.readthedocs.io/en/develop/style-guide.html#contract-and-library-names